projects/plugins/jetpack/extensions/blocks/ai-assistant/components/quota-exceeded-message/index.tsx

```diff
@@ -226,13 +226,18 @@ const FairUsageNotice = () => {
 };
 
 const QuotaExceededMessage = props => {
-	const { upgradeType, currentTier } = useAiFeature();
+	const { upgradeType, currentTier, tierPlansEnabled } = useAiFeature();
 
 	// Return notice component for the fair usage limit message, on unlimited plans.
 	if ( currentTier?.value === 1 ) {
 		return <FairUsageNotice />;
 	}
 
+	// Tiered plans when tier plans are disabled behave like unlimited plans.
+	if ( ! tierPlansEnabled && currentTier?.value > 1 ) {
```

Comment:
We may not need this check after we do the backend change to consider tiered plans as unlimited, since they will return the current tier value === 1.